### PR TITLE
fix(ui): Fix `MessageInput` textfield enabling behaviour

### PR DESCRIPTION
**Pull request by jiahan-wu**

### CODE REVIEW

**Comment by xsahil03x** [comment]
Hey @jiahan-wu, thanks for the fix 😄 
Can we do this instead?
```dart
 if (pickedFile != null) {
      final bytes = await pickedFile.readAsBytes();
      file = AttachmentFile(
        size: bytes.length,
        path: pickedFile.path,
        bytes: bytes,
      );
  }
```

and then it will be automatically called on line [1900](https://github.com/GetStream/stream-chat-flutter/pull/624/files#diff-ab9bbd9912f791b2dcaa78c30aa5a75289f7dce3a247f6dc3814067926f3bbc7R1900)

**Comment by jiahan-wu** [comment]
@xsahil03x Sure.😄

File: packages/stream_chat_flutter/CHANGELOG.md [hunk 1]
```diff
@@ -28,6 +28,10 @@ Here's the full naming breakdown:
 * `MessageTheme` is now `MessageThemeData`
 * `UserListViewTheme` is now `UserListViewThemeData`
 
+🐞 Fixed
+
+- Fixed `MessageInput` textField behaviour
```
Comment (xsahil03x) [inline packages/stream_chat_flutter/CHANGELOG.md:33]:
```suggestion
- Fixed `MessageInput` textField case where `input` is not enabled if the file picked from the camera is null.
```

**Review by xsahil03x — APPROVED** [review]
LGTM

**Pull request merged**